telosnetwork/telos-wallet
# 287 display aggregate token balance

# Fixes #287

## Description
see issue

## Test scenarios

1. login with account with more than one fiat balance token
2. see sum of fiat values represented in header on wallet page
3. login with empty account (or account with no tokens having fiat values)
4. see $0.00

## Checklist:
